examples/poc/basic_deployment/README.md

```diff
@@ -12,9 +12,21 @@ This deployment consists of:
 5. Creation of a new “Demo DB” - RDS MySQL
 6. Auto configuration of the RDS MySQL to enable native audit
 7. Onboard the DB to DSF Hub
-8. There is an option to create also a RDS MsSQL with audit configured and with synthetic data on it. In order to do so, use the variable 'db_types_to_onboard' and specify which DBs to create and onboard.
-For example, in order to create both 'RDS MySQL' and 'RDS MsSQL', run the following:
-```bash
-  terraform apply -auto-approve -var 'db_types_to_onboard=["RDS MySQL", "RDS MsSQL"]'
+8. There is an option to create also a RDS MsSQL with audit configured and with synthetic data on it. In order to do so, use the variable 'db_types_to_onboard' and specify which DBs to create and onboard.</br>
+    For example, in order to create both 'RDS MySQL' and 'RDS MsSQL', run the following:
+    ```bash
+      terraform apply -auto-approve -var 'db_types_to_onboard=["RDS MySQL", "RDS MsSQL"]'
    ```
-For more details, go to the rds-mssql-db module
\ No newline at end of file
+    For more details, go to the rds-mssql-db module
+
+## Default Example
+```bash
+terraform apply -auto-approve
+```
+
+## Customized Example
+The default example contains variables with default values. In order to customize the variables, you can use the following for example:
```

Comment:
The word 'example' is used with two different meanings in this sentence. I suggest: "you can use the following for instance:"